packages/rocketchat-custom-sounds/client/admin/soundInfo.js

```diff
@@ -90,15 +90,15 @@ Template.soundInfo.onCreated(function() {
 
 	this.autorun(() => {
 		const data = Template.currentData();
-		if (data.clear != null) {
+		if (data != null && data.clear != null) {
```

Comment:
I think you can change that to `if (data && data.clear !== null)` :)

Reply:
Sure, thank you for your comment.